scripts/globals/GlobalVariables.gd

```diff
@@ -7,6 +7,7 @@ var encryption_key := "https://youtu.be/OSkfONBg_QU?!?!".to_ascii_buffer();
 var level : int = 1
 # Arrays of doors stored at their door bitflag index
 # Every door which only has a bottom door will be stored at index [0b0001] so index [1]
+var playlists : Dictionary
```

Comment:
Might change where it is positioned in the file

Reply:
I think it's ok to put it here because it's a global settings